packages/es-dev-server/src/create-server.js

```diff
@@ -36,7 +47,46 @@ export function createServer(cfg, fileWatcher = chokidar.watch([])) {
       allowHTTP1: true,
```

Comment:
I was expecting this one to have to be false to allow connecting to the redirect server

Reply:
The application server can keep accepting HTTP1 and HTTP2 connections like it was before.

The new TCP server accepts anything and just forwards requests to either the redirect server or the application server, the protocols that they accept have no impact on this.